tokio-trace/src/lib.rs

```diff
@@ -352,6 +355,67 @@ pub struct Parents<'a> {
     next: Option<&'a SpanData>,
 }
 
+// ===== impl Meta =====
+impl<'a> Meta<'a> {
+    pub fn new_span(
```

Comment:
Same thing—doc comments would be help.

Reply:
Added in 34f0c3e60cb24b402bf17078ff75e9465cfb1b15